MediaBrowser.Model/MediaBrowser.Model.csproj

```diff
@@ -12,6 +12,8 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors Condition=" '$(Configuration)' == 'Release' ">true</TreatWarningsAsErrors>
+    <Nullable>enable</Nullable>
+    <LangVersion>latest</LangVersion>
```

Comment:
can we use explicit number? for reproducibility

Reply:
That would require changes when a new version comes out

Reply:
Yes. But that's okay! You'd have to change this value rarely, and reproducibility is absolutely worth it IMHO.

Reply:
`reproducibility`? Language version only impacts syntax, either it compiles or it doesn't. The generated IL won't change

Reply:
Oh, okay. Stay lazy 😝